addon/errors.js

```diff
@@ -4,12 +4,13 @@ const {
   A: emberArray,
   Object: EmberObject,
   get,
-  set
+  set,
+  run
 } = Ember;
 
 export default EmberObject.extend({
   unknownProperty(property) {
-    set(this, property, emberArray());
-    return get(this, property);
+    run.once(() => { set(this, property, emberArray()); });
```

Comment:
This seems not to be bug in this module but in your application. It should be your responsibility to avoid this. Try to concentrate to use no observers for initializing properties and use ember component hooks instead.